service/models/order.py

```diff
@@ -31,8 +31,11 @@ def serialize(self):
     def deserialize(self, data):
         """Populates an Order from a dictionary"""
         try:
+            self.id = data["id"]
```

Comment:
We shouldn't check "order.id" and "item.id" during deserialization. 
When a user sends a create request via a URL, we need to deserialize the user's request into an object. While, the user do not know the object's ID in advance because it hasn't been created yet XD.

[Professor's example account.py line72](https://github.com/nyu-devops/sample-accounts/blob/master/service/models/account.py)
[Professor's example test_account.py line 216](https://github.com/nyu-devops/sample-accounts/blob/master/tests/test_account.py)